extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/connection/ModeledConnection.java

```diff
@@ -117,12 +117,18 @@
     public static final String MAX_CONNECTIONS_PER_USER_NAME = "max-connections-per-user";
 
     /**
+     * The connection weight for weighted algorithms.
+     */
+    public static final String CONNECTION_WEIGHT = "weight";
+
+    /**
      * All attributes related to restricting user accounts, within a logical
      * form.
      */
     public static final Form CONCURRENCY_LIMITS = new Form("concurrency", Arrays.<Field>asList(
         new NumericField(MAX_CONNECTIONS_NAME),
-        new NumericField(MAX_CONNECTIONS_PER_USER_NAME)
+        new NumericField(MAX_CONNECTIONS_PER_USER_NAME),
+        new NumericField(CONNECTION_WEIGHT)
```

Comment:
Perhaps this should go in a different `Form`? As is, it will be rendered as one of several fields in a group labeled "Concurrency Limits", which is not what this relates to.

This would also help clean up the wording of the field itself, currently "Connection Weight for Load Balancing", as locating the field within a section titled "Load Balancing" (or similar) would remove the need to further qualify "Connection Weight" with the phrase "for Load Balancing".